File: app/src/main/java/com/mxt/anitrend/base/custom/view/text/SeriesTypeView.java
package com.mxt.anitrend.base.custom.view.text;

import android.content.Context;
import androidx.databinding.BindingAdapter;
import android.util.AttributeSet;

import com.mxt.anitrend.R;

public class SeriesTypeView extends SingleLineTextView {


    public SeriesTypeView(Context context) {
        super(context);
    }

    public SeriesTypeView(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    public SeriesTypeView(Context context, AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }

    @BindingAdapter("seriesType")
    public static void setSeriesType(SeriesTypeView view, String seriesType) {
        String attribute = view.getContext().getString(R.string.title_series_type);
        view.setText(String.format("%s %s", attribute, seriesType));
    }
}

File: app/src/main/java/com/mxt/anitrend/base/custom/view/text/SeriesTypeView.kt
package com.mxt.anitrend.base.custom.view.text

import android.content.Context
import android.util.AttributeSet
import com.mxt.anitrend.R

class SeriesTypeView @JvmOverloads constructor(
    context: Context,
    attrs: AttributeSet? = null,
    defStyleAttr: Int = 0
) : SingleLineTextView(context, attrs, defStyleAttr) {

    companion object {
        @JvmStatic
        fun setSeriesType(view: SeriesTypeView, seriesType: String?) {
            val attribute = view.context.getString(R.string.title_series_type)
            view.text = String.format("%s %s", attribute, seriesType)
        }
    }
}
